src/metalperformanceshaders.cs

```diff
@@ -1083,9 +1455,32 @@ interface MPSMatrixDescriptor {
 		[Static]
 		[Export ("rowBytesFromColumns:dataType:")]
 		nuint GetRowBytes (nuint columns, MPSDataType dataType);
+
+		[TV (11, 0), iOS (11, 0)]
+		[Export ("matrices")]
+		nuint Matrices { get; }
+
+		[TV (11, 0), iOS (11, 0)]
+		[Export ("matrixBytes")]
+		nuint MatrixBytes { get; }
+
+		[TV (11,0), iOS (11,0)]
+		[Static]
+		[Export ("matrixDescriptorWithRows:columns:rowBytes:dataType:")]
+		MPSMatrixDescriptor GetMatrixDescriptor (nuint rows, nuint columns, nuint rowBytes, MPSDataType dataType);
+
+		[TV (11,0), iOS (11,0)]
+		[Static]
+		[Export ("matrixDescriptorWithRows:columns:matrices:rowBytes:matrixBytes:dataType:")]
+		MPSMatrixDescriptor GetMatrixDescriptor (nuint rows, nuint columns, nuint matrices, nuint rowBytes, nuint matrixBytes, MPSDataType dataType);
+
+		[TV (11,0), iOS (11,0)]
+		[Static]
+		[Export ("rowBytesForColumns:dataType:")]
+		nuint GetRowBytesForColumns (nuint columns, MPSDataType dataType);
```

Comment:
I had to leave the ForColumns on this one because there's already another GetRowBytes method with the same parameters in the class:
```
                [Static]
		[Export ("rowBytesFromColumns:dataType:")]
		nuint GetRowBytes (nuint columns, MPSDataType dataType);
```

The only difference in the two selectors is From vs For, with the same parameter types for both.  I'm happy to use a different name if you have a better suggestion though.  Or, looking back, should I keep the FromColumns on the other one as well to help differentiate which is which?

Reply:
Ugg. That is going to be a tad ugly.

Maybe we should obsolete the unsuffixed (since it is totally ambiguous which it is) and have two with suffixes?

@spouliot thoughts?